README.rst

```diff
@@ -123,12 +123,9 @@ server with hot reload using::
 Switching between the webpack dev server and serving from ``static/`` requires a
 restart of your Django dev server to pick up the changed file paths.
 
-
 You must also configure Solr and install the configSet found under ``solr_conf``.
 If a core does not exist, and the configSet is installed correctly, an appropriate
-core will be made for you.
```

Comment:
why does this feel like "If you do not have a lawyer, one will be provided for you"

Reply:
😆 IDK! but this part will go away in my next PR anyway